oc-chef-pedant/spec/api/policies/complete_endpoint_spec.rb

```diff
@@ -804,7 +804,7 @@ def rm(assoc)
             end
           end
 
-          context "when the name is close to the maximum size" do
+          context "when the name is close to the maximum size", :chef_zero_quirks do
 
             # On Chef Zero, some backends will append `.json` to a file name,
             # which can exceed the common limit of 255 characters.
```

Comment:
I thought the intention here is that Chef Zero will filter out `:chef_zero_quirks` in its pedant runs, right? So it seems to me that either we can filter this test out, in which case we might as well change the size to erchef's maximum of 255, or not filter it out and use the zero-friendly value of 250, yeah? FWIW, the other maximum size tests below are for different properties (policy_name vs. revision_id vs. cookbook identifier).